lib/defs.js

```diff
@@ -56,6 +62,17 @@ const
           .replace(/"/g, '\\"') // escape double quotations
         return `"${ val }"`
       }
+    },
+    'TEXT': {
+        type: 'TEXT',
+            convert: val => {
+            val = (val || '')
+                .toString()
+                .replace(/\\/g, '\\\\') // escape backslashs
+                .replace(/"/g, '\\"') // escape double quotations
+                .replace(/[^\x00-\x7F]/g, "") // remove non ascii characters
```

Comment:
Oops, I'll remove this line, later.
We need to care about Unicode chars equally.